src/main/resources/assets/chisel/lang/de_de.lang

```diff
@@ -0,0 +1,2774 @@
+item.chisel.chisel_iron.name=Meißel
+item.chisel.chisel_diamond.name=Diamantmeißel
+item.chisel.chisel_hitech.name=iMeißel
+item.chisel.offsettool.name=Ender-Versatzstab
+
+chisel.spawnwither=Ein Meißel kann etwas gefährliches sein...
```

Comment:
Hey, this feature was actually removed, so uh...maybe worry about something else? :stuck_out_tongue: